WalletWasabi.Documentation/WasabiSetupRegtest.md

```diff
@@ -94,18 +96,17 @@ Todo:
 1. Go to `WalletWasabi\WalletWasabi.Fluent.Desktop` folder.
 2. Open the command line and run the Wasabi Client with:
 `dotnet run --no-build`
-3. Go to Tools/Settings and set the network to RegTest
+3. Go to Settings/Bitcoin and set the network to RegTest
 4. Close Wasabi and restart it with:
 `dotnet run --no-build`
 5. Generate a wallet in Wasabi named: R1.
-6. Generate a receive address in Wasabi, now go to Bitcoin Core gui to the Send tab.
+6. Generate a receive address in Wasabi, now go to Bitcoin Knots to the Send tab.
 7. Send 1 BTC to that address.
-8. Open another Wasabi instance from another command line:
-`dotnet run --no-build`
-9. Generate a wallet in Wasabi named: R2.
-10. Generate a receive address in Wasabi, now go to Bitcoin Core gui to the Send tab.
-11. Send 1 BTC to that address.
-12. Now in both instance go to CoinJoin tab and enqueue. CoinJoin should happen.
-13. If you see Waiting for confirmation in the Wasabi CoinList you can generate a block in Bitcoin Core to continue coinjoining.
+8. Generate a wallet in Wasabi named: R2.
+9. Generate a receive address in Wasabi, now go to Bitcoin Knots to the Send tab.
+10. Send 1 BTC to that address.
+11. Now let the coinjoin happen automatically in both instances.
```

Comment:
`instances` to `wallets`? @yahiheb?

Reply:
"Instances" no longer makes senses. I've replaced by "wallets".